common/src/main/java/com/wynntils/models/quests/QuestModel.java

```diff
@@ -106,11 +112,12 @@ public List<List<StyledText>> getDialogueHistory() {
     }
 
     public void startTracking(QuestInfo questInfo) {
-        CONTAINER_QUERIES.toggleTracking(questInfo);
+        Models.Content.startTracking(
+                questInfo.getName(), questInfo.isMiniQuest() ? ContentType.MINI_QUEST : ContentType.QUEST);
     }
 
     public void stopTracking() {
-        McUtils.sendCommand("tracking");
+        Models.Content.stopTracking();
```

Comment:
Did this break? I would assume the command to be faster...

Reply:
Yes, the command stopped working. :-(